Classes/Command/TransferCommandController.php

```diff
@@ -38,7 +38,7 @@ public function transferCommand()
 
         $success = true;
 
-        $repository = $this->objectManager->get('\EWW\Dpf\Services\Transfer\FedoraRepository');
+        $repository = $this->objectManager->get('EWW\Dpf\Services\Transfer\FedoraRepository');
```

Comment:
Removing the leading backslash ist good, as it won't work with 7.6 otherwise. The other backslashes should be escaped. Or.. use FedoraRepository::class instead.

All this instantiations will be fixed with pull-request https://github.com/kitodo/kitodo-publication/pull/89.

I'll propose to first merge #89 and to not touch these lines here.